src/SearchResults.js

```diff
@@ -15,12 +25,23 @@ const SearchResults = ({ results }) => {
           <th scope="col">Check in date</th>
           <th scope="col">Check out date</th>
           <th scope="col">nights</th>
+          <th scope="col">Profiles</th>
+          <th>
+      
+          </th>
         </tr>
       </thead>
       <tbody>
-        {results.map(booking => {
+      
+        
+        {props.results.map(booking => {
+          
+              
           return (
-            <tr>
+          
+         <tr  >
+         
+          
               <td>{booking.id}</td>
               <td>{booking.title}</td>
               <td>{booking.firstName}</td>
```

Comment:
Nice idea here, although you could go one step further and make everything in that `return` it's own `TableRow` component